pkg/cli/config_check_test.go

```
@@ -0,0 +1,471 @@
+		Output:      &bytes.Buffer{},
+	})
+	if err != nil {
+		t.Fatalf("remediateProjectConfig() error = %v", err)
+	}
+	if changed {
+		t.Fatal("changed = true, want clean no-op")
+	}
+}
+
+func TestAWSConfigRemediationMismatchDoesNotWrite(t *testing.T) {
+	root, cfg, inspection := setupConfigCheckProject(t)
```

golangci-lint (2.12.2)

[error] ineffectual assignment to inspection

(ineffassign)